rrdd/rrdd_server.ml

```diff
@@ -471,7 +472,7 @@ module Plugin = struct
       | e ->
         incr_skip_count uid plugin; (* increase skip count *)
         let log e =
-          warn "Failed to process plugin: %s (%s)"
+          info "Failed to process plugin: %s (%s)"
```

Comment:
Might be worth changing 'plugin' for 'file' to make the message less confusing.

Reply:
I added `metrics file` here to make it more explicit.